service.autosubs/addon.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="service.autosubs"
+    name="AutoSubs" version="1.1.0" provider-name="amet,schumi2004">
```

Comment:
this version needs to be higher than the ones in lower branches so that users can get the update when upgrading kodi. Please confirm

Reply:
keep the upstream source but add yourself as an author, so that we can "track" you when you update this addon in the future. Otherwise, it'll be the same store again :)

Reply:
it is. There's only 1.0.0  in Gotham. Did not find any newer